.travis.yml

```diff
@@ -70,9 +70,14 @@ env:
   - MATRIX_EVAL="CC=gcc-6 && CXX=g++-6"
 
 stages:
-  #- Build
-  - Test
-  - Build Compatibility
+  # - name: Build
+  #   if: type != cron
+  - name: Test
+    if: type != cron
```

Comment:
Don't know that it is necessary to not run them during the cron jobs, but if we can avoid using jobs when we can, I thought we should